nodejs18.x/full-stack/{{cookiecutter.project_name}}/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Should this .gitignore file include more files/dirs?

At minimum, I think `.aws-sam` should be ignored too, as this dir is created by running SAM CLI and should not be version controlled.

Reply:
I believe this file still needs to be updated.